src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -7,9 +7,14 @@
  */
 'use strict';
 
-// Polyfill in Firefox.
-// See https://blog.mozilla.org/webrtc/getdisplaymedia-now-available-in-adapter-js/
-if (adapter.browserDetails.browser == 'firefox') {
+const sharingPreference = document.getElementById('preference');
+
+if (adapter.browserDetails.browser === 'chrome' && adapter.browserDetails.version >= 107) {
+  // See https://developer.chrome.com/docs/web-platform/screen-sharing-controls/
```

Comment:
I'd prefer not to promote this pattern. How about simply letting the `select` always visible and add a link to the article or MDN later?

Reply:
this pattern is the sad reality (also here in the samples) unless there is a way to feature detect :-(